inductiva/fluids/simulators/dualsphysics.py

```diff
@@ -19,11 +18,10 @@ def run(
         self,
         input_dir: types.Path,
         sim_config_filename: str,
-        output_dir: Optional[types.Path] = None,
         device: Literal["gpu", "cpu"] = "cpu",
         resource_pool_id: Optional[UUID] = None,
         run_async: bool = False,
-    ) -> Union[pathlib.Path, Task]:
```

Comment:
Why remove the type annotation? This now always returns a task.

Reply:
And inherits from the Simulator which has that type annotation, so I thought to remove it here, since the import of tasks was only for that.

Tell me how you feel about this?

Reply:
I'd prefer to keep the type annotation, to be able to inspect the return type on IDEs, Jupyter Notebooks, etc.

Reply:
Alright!